fotoobo.yaml.sample

```diff
@@ -58,3 +55,16 @@ logging:
 #        host: localhost
 #        port: 514
 #        protocol: UDP   # UDP or TCP
+
+
+# Configure the Hashicorp Vault service
+# Instead of storing credentials in the inventory file you may use VAULT as a placeholder. All asset
+# attributes that are VAULT will be retreived from the Hashicorp Vault service specified here.
+# The Hashicorp Vault service will use approle login to get the data.
+vault:
+    url: https://vault.local
+    namespace: vault_namespace
+    data_path: /v1/kv/data/fotoobo
+    role_id: ...
+    secret_id: ...
+    token_file: ~/.cache/token.key
```

Comment:
Possibly add a comment that this setting is optional and may pose a security risk

Reply:
Added the following info:

    # This section is optional and may pose a security issue. In future versions we may also read the
    # role_id and secret_id from the system environment.